config/locales/en.yml

```diff
@@ -79,6 +79,29 @@ en:
   update: "Update"
   updated: "Updated"
 
+  general:
```

Comment:
This PR has already been reviewed and merged. I'd suggest letting this one
be. Putting all the general ones in the top level scope also could lead to
collisions.

On Tue, Feb 9, 2016 at 3:33 PM, Adam Konner <notifications@github.com>
wrote:

> In config/locales/en.yml
> <https://github.com/sassafrastech/madeline_system/pull/47#discussion_r52369140>
> :
>
> > @@ -79,6 +79,29 @@ en:
> >    update: "Update"
> >    updated: "Updated"
> >
> > +  general:
>
> Let's not nest the "general" ones—it just adds unnecessary code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sassafrastech/madeline_system/pull/47/files#r52369140>
> .
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech

Resident, Touchstone Cohousing
touchstonecohousing.org